ieee/template.typ

```diff
@@ -22,14 +22,17 @@
   // works.
   bibliography-file: none,
 
+  // Font to be used throughout the document.
+  font: "Times New Roman",
```

Comment:
This can't be the default font because it's a proprietary font and the web app doesn't have it. STIX Two Text is the free Times New Roman alternative.

Reply:
Alrighty, makes sense - I have changed that back to `STIX Two Text`.